mypy/build.py

```diff
@@ -891,7 +891,7 @@ def find_cache_meta(id: str, path: str, manager: BuildManager) -> Optional[Cache
         return None
 
     # Ignore cache if generated by an older mypy version.
-    if (m.version_id != manager.version_id
+    if ((m.version_id != manager.version_id and not manager.options.skip_version_check)
```

Comment:
Minorest nit: The `or` clauses here break over a line, I think for consistency it'd be nice to do so with the `and`.